src/services/rhsm/rhsmServices.js

```diff
@@ -1883,7 +1890,7 @@ const getInstancesInventory = (id, params = {}, options = {}) => {
  *           "physical_capacity": 2,
  *           "virtual_capacity": 2,
  *           "total_capacity": 4,
- *           "uom": "Cores",
+ *           "metric_id": "Cores"
```

Comment:
Even though this code is commented out, these changes are missing a comma at the end of the line.

Reply:
Trailing commas are a linting preference only ever. Please see linting rules.

Reply:
~Trailing commas are optional only ever, please see linting rules.~ But ah, that's what you're saying. Good catch we'll update the mock

Reply:
I don't want to be nitpicking, but I believe this is not a case of trailing commas since it is not the last item in the object/dictionary. There is `"has_infinite_quantity": false` after this line.

Reply:
Correct, i noted my error